sdk/ml/azure-ai-ml/setup.py

```diff
@@ -86,6 +86,8 @@
         "azure-common<2.0.0,>=1.1",
         "typing-extensions<5.0.0",
         "opencensus-ext-azure<2.0.0",
+        "opencensus-ext-logging",
+        "azureml-telemetry<=1.53.0",
```

Comment:
Would be great to check how heavy this dependency is

Reply:
Which one? Telemetry or opencensus?

Telemetry actually could be removed...

Reply:
I've removed it from all 3 packages.

Reply:
azureml-telemetry brings in azureml-core, which will be a problem down the road. Thank you for removing it.